components/eam/bld/namelist_files/namelist_definition.xml

```diff
@@ -4201,6 +4201,18 @@ number of so4(+nh4) monolayers needed to "age" a carbon particle
 Default: Set by namelist
 </entry>
 
+<entry id="dp_cut_accum_rename" type="real"  category="scavenging"
+       group="phys_ctl_nl" valid_values="" >
+accum mode aerosol diameter threshold for renaming to strat coarse mode
+Default: Set by namelist
+</entry>
+
+<entry id="dp_xferall_thresh_accum_rename" type="real"  category="scavenging"
+       group="phys_ctl_nl" valid_values="" >
+accum mode aerosol diameter threshold for renaming to strat coarse mode
```

Comment:
@mingxuanwupnnl . Can you make the descriptions for these two namelist variables distinguishable?